src/lib/support/tests/TestVariant.cpp

```diff
@@ -238,66 +237,35 @@ void TestVariantCompare(nlTestSuite * inSuite, void * inContext)
     v3.Set<Pod>(5, 10);
     v4.Set<Pod>(5, 11);
 
-    NL_TEST_ASSERT(inSuite, (v0 == v0));
-    NL_TEST_ASSERT(inSuite, !(v0 == v1));
-    NL_TEST_ASSERT(inSuite, !(v0 == v2));
-    NL_TEST_ASSERT(inSuite, !(v0 == v3));
-    NL_TEST_ASSERT(inSuite, !(v0 == v4));
-
-    NL_TEST_ASSERT(inSuite, !(v1 == v0));
-    NL_TEST_ASSERT(inSuite, (v1 == v1));
-    NL_TEST_ASSERT(inSuite, !(v1 == v2));
-    NL_TEST_ASSERT(inSuite, !(v1 == v3));
-    NL_TEST_ASSERT(inSuite, !(v1 == v4));
-
-    NL_TEST_ASSERT(inSuite, !(v2 == v0));
-    NL_TEST_ASSERT(inSuite, !(v2 == v1));
-    NL_TEST_ASSERT(inSuite, (v2 == v2));
-    NL_TEST_ASSERT(inSuite, (v2 == v3));
-    NL_TEST_ASSERT(inSuite, !(v2 == v4));
-
-    NL_TEST_ASSERT(inSuite, !(v3 == v0));
-    NL_TEST_ASSERT(inSuite, !(v3 == v1));
-    NL_TEST_ASSERT(inSuite, (v3 == v2));
-    NL_TEST_ASSERT(inSuite, (v3 == v3));
-    NL_TEST_ASSERT(inSuite, !(v3 == v4));
-
-    NL_TEST_ASSERT(inSuite, !(v4 == v0));
-    NL_TEST_ASSERT(inSuite, !(v4 == v1));
-    NL_TEST_ASSERT(inSuite, !(v4 == v2));
-    NL_TEST_ASSERT(inSuite, !(v4 == v3));
-    NL_TEST_ASSERT(inSuite, (v4 == v4));
-}
-
-int Setup(void * inContext)
-{
-    return SUCCESS;
-}
-
-int Teardown(void * inContext)
-{
-    return SUCCESS;
+    EXPECT_TRUE(v0 == v0);
```

Comment:
Use `EXPECT_EQ` or `EXPECT_NE`.

Reply:
This does not seem to work for me when we have EXPECT_NE, I assume since the Variant Class is not implementing operator overloading for operator!=

Reply:
So I left this as is, I wrote to you on slack